docker/manage

```diff
@@ -28,24 +28,31 @@ SCRIPT_HOME="$( cd "$( dirname "$0" )" && pwd )"
 usage() {
   cat <<-EOF
 
-  Usage: $0 {start|stop|build|rm|lint}
+  Usage: $0 [command] [options]
 
-  Options:
+  Commands:
 
   build - Build the docker images for the project.
           You need to do this first, since the builds require
           a combination of Docker images.
 
-  start - Creates the application container from the built images
-          and starts the services based on the docker-compose.yml file.
+  up -  Creates the application container from the built images
+        and starts the services based on the docker-compose.yml file.
 
-          You can pass in a list of containers to start.
-          By default all containers will be started.
+        You can pass in a list of containers to start.
+        By default all containers will be started.
+
+  start - Same as up
+
+  logs - Display the logs from the docker compose run (ctrl-c to exit).
 
   stop - Stops the services.  This is a non-destructive process.  The containers
          are not deleted so they will be reused the next time you run start.
 
-  rm - Removes any existing application containers.
+  down - Brings down the services and removes the volumes (storage).  The containers
```

Comment:
The `down` command is destructive, containers and volumes are deleted.  Please update documentation.